clar2wasm/tests/wasm-generation/sequences.rs

```diff
@@ -43,6 +43,7 @@ proptest! {
     #![proptest_config(super::runtime_config())]
 
     #[test]
+    #[ignore]
```

Comment:
Any specific reason to ignore this test?

Reply:
https://github.com/stacks-network/clarity-wasm/issues/368